massa-sdk/src/lib.rs

```diff
@@ -642,18 +643,22 @@ impl RpcClientV2 {
 }
 
 fn http_client_from_url(url: &str, http_config: &HttpConfig) -> HttpClient<HttpBackend> {
-    let mut builder = HttpClientBuilder::default()
+    let builder = HttpClientBuilder::default()
         .max_request_size(http_config.client_config.max_request_body_size)
         .request_timeout(http_config.client_config.request_timeout.to_duration())
-        .max_concurrent_requests(http_config.client_config.max_concurrent_requests)
+        // .max_concurrent_requests(http_config.client_config.max_concurrent_requests)
```

Comment:
There isn't this config anymore? What's the impact?

Reply:
Unfortunately the field max_concurrent_requests is now private (default is set to: 256). I'll make a PR for jsonrpsee and in the mean time will document this comment.

Reply:
Perfect thanks ! I'll keep this comment unresolved to avoid forgetting

Reply:
we need to wait your PR to fix this before merge on main

Reply:
https://github.com/paritytech/jsonrpsee/pull/1470